recipes/dcmtk/all/conanfile.py

```diff
@@ -287,7 +296,7 @@ def package_info(self):
 
         def register_components(components):
             for target_lib, requires in components.items():
-                self.cpp_info.components[target_lib].set_property("cmake_target_name", target_lib)
+                self.cpp_info.components[target_lib].set_property("cmake_target_name", f"DCMTK::{target_lib}")
```

Comment:
I am a bit concerned about this change. It means that it will break existing users of this package, because their target names are being changed, isn't it? From https://support.dcmtk.org/redmine/projects/dcmtk/wiki/Howto_CMakeExports it seems that the bare target name (without ``DCMTK::``) was the expected one?

Reply:
Hm at least on our systems we've always been using `DCMTK::$TARGET` and this only changed after the switch to Conan v2

Reply:
I've reverted this for now